### ISSUE TITLE
Problem GnuPG validation in MinGW watch scripts

### ISSUE THREAD

**Issue by bastianeicher** [0install/apps#178]
`mingw64watch.py` seems to correctly import `Alexey.pubkey` but then exits with:

```
database http://repo.msys2.org/mingw/x86_64/mingw64.db.tar.gz invalid signature
                0, signature valid
```

@pmiess Could you take another look at this?

**Comment by pmiess** [comment]
pull request up

**Comment by bastianeicher** [comment]
Fixed via #179
